### PR TITLE
add GetSteteChannels

### PR DESCRIPTION
**Pull request by Jianzhuo**
add GetSteteChannels for graphql

### CODE REVIEW

File: packages/network-query/queries/network/statechannel.gql [hunk 1]
```diff
@@ -22,3 +22,21 @@ query GetPaygPlans($indexer: String!, $deploymentId: String!, $status: ChannelSt
     }
   }
 }
+
+query GetSteteChannels($status: ChannelStatus!) {
```
Comment (neo-sun) [inline packages/network-query/queries/network/statechannel.gql:26]: `stete` ? is `state` ?

File: packages/network-query/queries/network/statechannel.gql [hunk 2]
```diff
@@ -22,3 +22,21 @@ query GetPaygPlans($indexer: String!, $deploymentId: String!, $status: ChannelSt
...
+  stateChannels(filter: { status: { equalTo: $status } }) {
+    nodes {
+      id
+      indexer
+      consumer
+      status
+      total
+      spent
```
Comment (neo-sun) [inline packages/network-query/queries/network/statechannel.gql:34]: missing the price

**Comment by HZHAIX** [comment]
Add test case for this query, can re-run the test CI once host project is 100% synced

**Review by neo-sun — APPROVED** [review]

**Comment by HZHAIX** [comment]
Will leave this PR here and run the test again after https://explorer.subquery.network/subquery/subquery/kepler-testnet-subql-project?stage=true 100% synced

**Pull request merged**